public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -106,11 +106,16 @@ export function ChannelSubForm<R extends ChannelValues>({
       sortBy(notifiers, ({ dto, meta }) => [meta?.order ?? 0, dto.name])
         // .notifiers.sort((a, b) => a.dto.name.localeCompare(b.dto.name))
         .map<SelectableValue>(({ dto: { name, type }, meta }) => ({
-          label: name,
+          // @ts-expect-error ReactNode is supported
+          label: (
+            <Stack alignItems="center" gap={1}>
+              {name}
+              {meta?.badge}
+            </Stack>
+          ),
```

Comment:
Can we fix the error rather than `ts-expect`ing it? Or is there a shortcoming in our types somewhere?

Reply:
I wanted to avoid the domino of changes because this type is widely used and is defined in grafana-data
![image](https://github.com/user-attachments/assets/a76fc4cb-d76e-4a61-888d-357b7408b0d8)

And we actually need to use JSX instead of string in here. We have couple of such places in IRM too, probably worth to fix it in the source but I'd do that maybe in separate PR? Wdyt?